### PR TITLE
ADD ignoreType (for extra locs) metadata feature

### PR DESCRIPTION
**Pull request by fgalan**
Issue #4032

### CODE REVIEW

File: doc/manuals/user/ngsiv2_implementation_notes.md [hunk 1]
```diff
@@ -144,6 +144,43 @@ From "Geospatial properties of entities" section at NGSIv2 specification:
...
+  "serviceArea": {
+    "value": {
+      "type": "Polygon",
+      "coordinates": [ [ [-3.69807, 40.49029 ], [ -3.68640, 40.49100], [-3.68602, 40.50456], [-3.71192, 40.50420], [-3.69807, 40.49029 ] ] ]
+    },
+    "type": "geo:json",
+    "metadata": {
+      "value": true,
+      "type": "Boolean"
+    }
```
[the remark this answers is not in the document]
Comment (fgalan) [reply doc/manuals/user/ngsiv2_implementation_notes.md]: Fixed in d4311e7

**Comment by fgalan** [comment]
`extraLoc` is an ugly name... try to improve it

**Review by mapedraza — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
@fisuda this PR modifies .md documentation. It would be great if you could sync changes in the Japanese translation, please. Thanks!

**Comment by fisuda** [comment]
I sent the PR #4053.
Thanks.

**Comment by cesarjorgemartinez** [comment]
LGTM

**Pull request merged**